OB-185 User API

zesilvapro opened the pull request:
- Adicionei API ...

zesilvapro committed:
.

HenriqueMoniz reviewed (approved):
Tudo partido mas pode ir para a frente

tentone commented on Backend-Roadmap/.gitignore:
```diff
@@ -0,0 +1,2 @@
+.vs/*
```
Check the gitignore to include all temporary files

zesilvapro replied:
.ignore actualizado

jl-eqsdigital commented on Backend-Roadmap/Program.cs:
```diff
@@ -13,6 +13,8 @@ public static void Main(string[] args)
             // Add services to the container.
             builder.Services.AddControllers();
 
+ // teste
```
remove

zesilvapro replied:
deleted

zesilvapro committed:
Remove temp files

zesilvapro committed:
.gitignore update

zesilvapro committed:
..

zesilvapro committed:
..

zesilvapro committed:
Update .ignore

zesilvapro committed:
UsersController

Update on connection string due to workstation switch and update to implement CRUD on Users table

zesilvapro commented:
- [X] .gitignore 
- [X]  User table CRUD implemented
![SwaggerControl](https://github.com/zesilvapro/RoadmapEQS/assets/151776359/23fc58e2-9302-453d-a7af-5519281fa826)

jl-eqsdigital commented on Backend-Roadmap/Controllers/UsersController.cs:
```diff
@@ -115,6 +115,66 @@ public ActionResult RegisterUser([FromBody] Users newUser)
+
+        [HttpPut]
+        [Route("UpdateUser/{userId}")]
+        public ActionResult UpdateUser(int userId, Users updatedUser)
```
Usually it's
```
public ActionResult UpdateUser(int userId, [FromBody] Users userToUpdate)
```
try naming the variables so we know what they mean

jl-eqsdigital replied:
Try also programming asynchronously with async Task<T>  and  await

zesilvapro replied:
Updated

zesilvapro committed:
UpdateUser changed

zesilvapro committed:
Async change
